src/GeoPing.Api/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api
+{
+    public static class Constants
```

Comment:
still hardcoded in static manner, can we use Configuration for this?

Reply:
In progress